torchdynamo/config.py

```diff
@@ -1,3 +1,4 @@
+import logging
```

Comment:
I'm curious does `import torch` also bring in its own logger? I'm worried that potentially some PyTorch logs could be disabled if a user enables dynamo logging